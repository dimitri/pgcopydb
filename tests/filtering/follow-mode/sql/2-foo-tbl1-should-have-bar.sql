-- foo.tbl1 should have bar in desc_text
select
    desc_text
from
    foo.tbl1
where
    desc_text = 'bar';
