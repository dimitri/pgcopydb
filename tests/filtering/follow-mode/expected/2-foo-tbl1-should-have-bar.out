-[ RECORD 1 ]--
desc_text | bar

